### PR TITLE
Add SX1268 modules support for linux-native

### PR DESCRIPTION
**Pull request by SannySannyman**
The code is tested with E22 400M33S module, but should work for other SX1268-based modules, as it reuses the existing radio interface code.

### CODE REVIEW

**Comment by mverch67** [comment]
bin/config-dist.yaml
 -:-  fmt  Incorrect formatting, autoformat by running '**trunk fmt**'  prettier

File: src/main.cpp [hunk 1]
```diff
@@ -768,6 +768,21 @@ void setup()
                 LOG_INFO("SX1280 Radio init succeeded, using SX1280 radio\n");
             }
         }
+    } else if (settingsMap[use_sx1268]) {
+        if (!rIf) {
+            LOG_DEBUG("Attempting to activate sx1268 radio on SPI port %s\n", settingsStrings[spidev].c_str());
+            LockingArduinoHal *RadioLibHAL = new LockingArduinoHal(SPI, spiSettings);
```
Comment (jp-bennett) [inline src/main.cpp:774]: I believe this now needs to use *LoraSPI to avoid potential conflicts with the display or touchscreen.

**Review by jp-bennett — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 1 above)
Comment (SannySannyman) [reply src/main.cpp]: Fixed.

**Comment by SannySannyman** [comment]
> bin/config-dist.yaml -:- fmt Incorrect formatting, autoformat by running '**trunk fmt**' prettier

Done.

**Review by jp-bennett — APPROVED** [review]
Looks good to me

**Pull request merged**